src/Lkl.cc

```diff
@@ -491,6 +495,7 @@ void Lkl::FindGLowAndGUpp(Double_t& glow,Double_t& gupp,Bool_t centerAtZero)
       Double_t save  = gupp;      
       gupp+=(gupp<0? 2*TMath::Abs(gupp) : TMath::Abs(gupp));
       cout << "Lkl::FindGLowAndGUpp (" << GetName() << ") Message: -2logL(gupp) = " << lklval << ", gupp rised from " << save << " to " << gupp <<" (fLklMin = " << fgmin << ")"<< endl;
+      if(lklval >= 1e99) break;
```

Comment:
As in the previous comment, this should never happen, right?

Reply:
I'm almost certain it does, I will try to find a past example of this.

Reply:
Same as before